charts/third-party/authup/templates/deployment.yaml

```
@@ -125,5 +138,17 @@ spec:
                   port: 3000
               failureThreshold: 3
               periodSeconds: 60
+        {{- if .Values.provisioning.enabled }}
+          volumeMounts:
+              -   name: provisioning
```

YAMLlint (1.37.1)

[warning] too many spaces after hyphen

(hyphens)